react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -282,21 +268,25 @@ class RemoteVideoMenuTriggerButton extends Component<Props> {
                 buttons.push(
                     <MuteButton
                         key = 'mute'
+                        onClick = { this.props.hidePopover }
```

Comment:
what if instead of modifying each button to support the hide popover functionality, we wrap them in a component that's responsible for doing that and allows the click to go through to its child components?

Reply:
Yeah, you're right.  I'll try to group the buttons that will hide popover and wrap them in a component. Did not think this through because there are buttons that should not close popover on click, but it's doable...

Reply:
well not necessarily to group them under one component, we can also have each one of them wrapped in a component if it's easier